pkg/acme/client/middleware/logger.go

```diff
@@ -73,6 +73,15 @@ func (l *Logger) FetchCert(ctx context.Context, url string, bundle bool) ([][]by
 	return l.baseCl.FetchCert(ctx, url, bundle)
 }
 
+func (l *Logger) FetchCertAlternatives(ctx context.Context, url string, bundle bool) ([][][]byte, error) {
+	l.log.V(logf.TraceLevel).Info("Calling FetchCertAlternatives")
+
+	ctx, cancel := context.WithTimeout(ctx, timeout)
+	defer cancel()
```

Comment:
Seems a bit weird to add a timeout in "logging" middleware, but I see that that is done elsewhere, so fine.

Reply:
This always has been a weird middleware... never liked it but it has it's reasons to be here